cedar-policy-formatter/src/pprint/fmt.rs

```diff
@@ -130,69 +130,51 @@ pub fn policies_str_to_pretty(ps: &str, config: &Config) -> Result<String> {
 
 #[cfg(test)]
 mod tests {
-    use super::*;
-    const TEST_CONFIG: &Config = &Config {
-        line_width: 40,
-        indent_width: 2,
-    };
-
-    #[test]
-    fn trivial_permit() {
-        let policy = r#"permit (principal, action, resource);"#;
-        assert_eq!(policies_str_to_pretty(policy, TEST_CONFIG).unwrap(), policy);
-    }
+    use insta::{assert_snapshot, glob, with_settings};
+    use std::fs;
 
-    #[test]
-    fn trivial_forbid() {
-        let policy = r#"forbid (principal, action, resource);"#;
-        assert_eq!(policies_str_to_pretty(policy, TEST_CONFIG).unwrap(), policy);
-    }
-
-    #[test]
-    fn action_in_set() {
-        let policy = r#"permit (
-        principal in UserGroup::"abc",
-        action in [Action::"viewPhoto", Action::"viewComments"],
-        resource in Album::"one"
-      );"#;
-        assert_eq!(
-            policies_str_to_pretty(policy, TEST_CONFIG).unwrap(),
-            r#"permit (
-  principal in UserGroup::"abc",
-  action in
-    [Action::"viewPhoto",
-     Action::"viewComments"],
-  resource in Album::"one"
-);"#
-        );
-    }
+    use super::*;
 
     #[test]
     fn test_format_files() {
-        use std::fs::read_to_string;
-        use std::path::Path;
-
         let config = Config {
             line_width: 80,
             indent_width: 2,
         };
-        let dir_path = Path::new(env!("CARGO_MANIFEST_DIR")).join("tests");
-        let pairs = vec![
-            ("test.cedar", "test_formatted.cedar"),
-            ("policies.cedar", "policies_formatted.cedar"),
-            ("is_policies.cedar", "is_policies_formatted.cedar"),
-        ];
-        for (pf, ef) in pairs {
-            // editors or cargo run try to append a newline at the end of files
-            // we should remove them for equality testing
-            assert_eq!(
-                policies_str_to_pretty(&read_to_string(dir_path.join(pf)).unwrap(), &config)
-                    .unwrap()
-                    .trim_end_matches('\n'),
-                read_to_string(dir_path.join(ef))
-                    .unwrap()
-                    .trim_end_matches('\n')
-            );
-        }
+
+        // This test uses `insta` to test the current output of the formatter
```

Comment:
Minor: Can we move these comments to a README file so that users or developers can quickly figure out how to use it.

Reply:
I'm personally more likely to look at the failing test case than the README, but I can add a note there as well